src/functionBased/pages/About.js

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+
+const About = (props) => {
+  console.log(props)
+  return <div>hello from about page</div>
+}
```

Comment:
Here I think the below 2 changes can be done-
- There should be no console.log files in your code in production, Always remove them before sending code to production.
- I think you can add more details about yourself in About page